app/components/Layout.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import Nav from '../components/Nav';
+import Head from 'next/head';
+import styles from '../styles/Layout.module.css';
+
+/**
+ * Page layout wrapper
+ *
+ * @param {*} props
+ * @returns {React.component}
+ */
+export default function Layout({ title, children }) {
+  return (
+    <>
+      <Nav />
+      <div className={styles.container}>
+        <Head>
+          <title>
+            {title ? `${title} | Startup Directory` : 'Startup Directory'}
+          </title>
+          <link rel="icon" href="/favicon.ico" />
```

Comment:
Should change the favicon to the founders logo for the future

Reply:
Agreed, but unrelated to this PR